src/OrchardCore.Modules/OrchardCore.Users/TwoFactorAuthenticationStartup.cs

```diff
@@ -105,6 +112,11 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddScoped<IDisplayDriver<TwoFactorMethod>, TwoFactorMethodLoginEmailDisplayDriver>();
         services.AddScoped<IDisplayDriver<ISite>, EmailAuthenticatorLoginSettingsDisplayDriver>();
     }
+
+    public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder routes, IServiceProvider serviceProvider)
+    {
+        routes.AddEmailSendCodeEndpoint<EmailAuthenticatorStartup>();
+    }
```

Comment:
```suggestion
    public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder routes, IServiceProvider serviceProvider)
        => routes.AddEmailSendCodeEndpoint<EmailAuthenticatorStartup>();
```

Reply:
I rather not. This was if someone wants to add more service they can without modifying the body again